<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protótipo de Reconhecimento Facial</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/profile.jpg</normaloff>img/profile.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>11</y>
      <width>45</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perfil</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>11</y>
      <width>198</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Captura de Webcamm</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>42</y>
      <width>505</width>
      <height>505</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>505</width>
      <height>505</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>505</width>
      <height>505</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="validarIndividuoButton">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>560</y>
      <width>501</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Validar Identificação</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="novoIndividuoTextEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>299</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Nome Indivíduo</string>
    </property>
   </widget>
   <widget class="QLabel" name="identidadeLabel">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>65</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Configurações</string>
    </property>
    <addaction name="adicionarNovoIndividuoAction"/>
    <addaction name="treinarIndividuosAction"/>
   </widget>
   <addaction name="menuConfigura_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="adicionarNovoIndividuoAction">
   <property name="text">
    <string>Novo Individuo</string>
   </property>
  </action>
  <action name="treinarIndividuosAction">
   <property name="text">
    <string>Treinar </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
